### PR TITLE
FISH-5724 Stopped throwing NamingException on PostConstruct method

### PR DESCRIPTION
**Pull request by kalinchan**
## Description

NamingException was thrown on the PostConstruct method.
### Blockers

N/A
### New tests

N/A
### Testing Performed

Built Payara
### Testing Environment

ubuntu 20.04 openjdk version "1.8.0_292" maven 3.6.3
## Documentation

N/A
## Notes for Reviewers

N/A

### CODE REVIEW

**Comment by kalinchan** [comment]
jenkins test

File: appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/service/FaultToleranceServiceImpl.java [hunk 1]
```diff
@@ -120,14 +110,18 @@
...
+        try {
+            events.register(this);
+            invocationManager = serviceLocator.getService(InvocationManager.class);
+            requestTracingService = serviceLocator.getService(RequestTracingService.class);
+            config = serviceLocator.getService(FaultToleranceServiceConfiguration.class);
+            InitialContext context = new InitialContext();
+            asyncExecutorService = (ManagedExecutorService) context.lookup(config.getManagedExecutorService());
+            delayExecutorService = (ManagedScheduledExecutorService) context.lookup(config.getManagedScheduledExecutorService());
+        } catch (NamingException namingException) {
+            throw new RuntimeException("FaultToleranceServiceImpl InitialContext creation failure", namingException);
```
Comment (Pandrex247) [inline appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/service/FaultToleranceServiceImpl.java:123]: I'd provide a more useful message
```suggestion
            throw new RuntimeException("Error initialising Fault Tolerance Service: could not perform lookup for configured managed-executor-service or managed-scheduled-executor-service.", namingException);
```

**Review by Pandrex247 — COMMENTED** [review]
Some comments

**Review by Pandrex247 — APPROVED** [review]

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by kalinchan** [comment]
jenkins test

**Pull request merged**